motion/motion_launch/launch/motion.launch

```diff
@@ -7,48 +7,31 @@
-      <node pkg="los_controller"      type="los_controller_node.py"     name="los_controller"   output="screen"/>
-      <node pkg="dp_controller"       type="dp_controller_node"         name="dp"               output="screen"/>
-    </group>
-
-      <!-- Thrust nodes -->
-    <group ns="$(arg ns_thrust)">
-      <node pkg="thruster_interface"  type="thruster_interface_node.py"   name="thruster_interface"/>
-      <node pkg="thruster_allocator"  type="thruster_allocator_node"      name="thruster_allocator"/>
-      <node pkg="mcu_interface"       type="mcu_interface_node"           name="mcu_interface"  if="$(eval type == 'real')"/> 
-    </group>
+  <!-- Guidance nodes -->
+  <group ns="$(arg ns_guidance)">
+    <node pkg="los_guidance" type="los_guidance_backstepping.py" name="los"                         if="$(eval type == 'real')"     />
+    <node pkg="los_guidance" type="los_guidance_backstepping.py" name="los"       output="screen"   if="$(eval type == 'simulator')"/>
+    <node pkg="dp_guidance"  type="dp_guidance.py"               name="dp"        output="screen"                                   />
+    <node pkg="interface"    type="guidance_interface.py"        name="interface" output="screen"                                   />
   </group>
 
+  <!-- Controller nodes -->
+  <group ns="$(arg ns_controller)">
+    <node pkg="los_controller"      type="los_controller_node.py"     name="los_controller"   output="screen"/>
+    <node pkg="dp_controller"       type="dp_controller_node"         name="dp"               output="screen"/>
+  </group>
 
-  <!-- ROV motion -->
-  <group if="$(eval control_mode == 'manual')">
+  <!-- Thrust nodes -->
+  <group ns="$(arg ns_thrust)">
+    <node pkg="thruster_interface"  type="thruster_interface_node.py"   name="thruster_interface"/>
+    <node pkg="thruster_allocator"  type="thruster_allocator_node"      name="thruster_allocator"/>
+    <node pkg="mcu_interface"       type="mcu_interface_node"           name="mcu_interface"  if="$(eval type == 'real')"/> 
+  </group>
 
-    <group ns="$(arg ns_joystick)">
-  
-      <!-- Joystick nodes -->
-      <node pkg="joy"                name="joystick_driver"    type="joy_node"              output="screen" args="_deadzone:=0.15 _autorepeat_rate:=100"/>
-      <node pkg="joystick_guidance"  name="joystick_guidance"  type="joystick_guidance.py"  output="screen"/>
-    </group>
 
-    <!-- Thrust nodes -->
-    <group ns="$(arg ns_thrust)">
-      <node pkg="thruster_interface"  type="thruster_interface_node.py"   name="interface"/>
-      <node pkg="vortex_allocator"    type="vortex_allocator_node"        name="allocator"/>
-      <node pkg="mcu_interface"       type="mcu_interface_node"           name="mcu_interface"  if="$(eval type == 'real')"/> 
-    </group>
+  <!-- Joystick nodes -->
+  <group ns="$(arg ns_joystick)">
+    <node pkg="joystick_guidance"  name="joystick_guidance"  type="joystick_guidance.py"  output="screen"/>
```

Comment:
What is the reason for keeping this separate and not together with the other guidance nodes?

Reply:
Might be an artifact from the previous ROV configuration that I implemented. 

Currently all incoming data for the guidance nodes goes through the guidance_interface node.  Should I change the current system to be "topside joystick" -> "guidance interface" -> "joystick guidance"?

Reply:
ok, then I think it should be placed together with the other guidance nodes. 

Regarding how the interface should be involved in this: I think "topside joystick" -> "guidance interface" -> "joystick guidance" is better. In the last meeting, we agreed that guidance_interface should activate a position hold mode whenever no other commands are given, so there definitely has to be a connection between joystick_guidance and guidance_interface. Merging joystick_interface with guidance_interface might simplify the implementation of this. Having only one "interface" in guidance would also be a less cluttered setup.